### PR TITLE
Fix reward module tokenID validation - Closes #7349 

### LINKED ISSUE #7349: Commander command blockchain:reset needs to be updated
### Expected behavior

Should clear all dbs and allow to re-rerun the node.

### Actual behavior

Error when re-rerunning the node:
```
Message: Invalid validator key.
 Trace: Error: Invalid validator key
```

### Steps to reproduce

Run node for few blocks. Run blockchain:reset. Then run the node again.

### Which version(s) does this affect? (Environment, OS, etc...)
development

### PR DESCRIPTION
**Pull request by mitsuaki-u**
### What was the problem?

This PR resolves #7349

### How was it solved?

Added schema validation for config

### How was it tested?

Added unit test

### CODE REVIEW

**Review by shuse2 — CHANGES_REQUESTED** [review]
I think in reward module `init` it should set the default value if it's not given.
Also, we should expect config comes as `hex string`, and convert to Buffer inside of `init` function

File: framework/src/modules/reward/module.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class RewardModule extends BaseModule {
 			throw new LiskValidationError(errors);
 		}
 		this._moduleConfig = (config as unknown) as ModuleConfig;
-		this._tokenIDReward = this._moduleConfig.tokenIDReward;
+		this._tokenID = Buffer.from(this._moduleConfig.tokenID, 'utf8');
```
Comment (shuse2) [inline framework/src/modules/reward/module.ts:71]:
```suggestion
		this._tokenID = Buffer.from(this._moduleConfig.tokenID, 'hex');
```

File: framework/src/modules/reward/module.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ export class RewardModule extends BaseModule {
 		await this._tokenAPI.mint(
 			context.getAPIContext(),
 			context.header.generatorAddress,
-			this._tokenIDReward,
+			this._tokenID,
```
Comment (shuse2) [inline framework/src/modules/reward/module.ts:105]: I think the variable name was aligned with other modules

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: framework/src/modules/reward/module.ts (hunk 2 above)
Comment (mitsuaki-u) [reply framework/src/modules/reward/module.ts:105]: After discussion will keep as is

**Review by shuse2 — APPROVED** [review]

**Pull request merged**